sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/Processor/DataProcessor.cs

```diff
@@ -105,9 +105,10 @@ public TestResults GetTestCaseResultData(TestResult? testResultSource)
             testCaseResultData.TestCombinationId = testCaseResultData.TestExecutionId; // TODO check
             testCaseResultData.TestId = testResultSource.TestCase.Id.ToString();
             testCaseResultData.TestTitle = testResultSource.TestCase.DisplayName;
-            var className = FetchTestClassName(testResultSource.TestCase.FullyQualifiedName);
-            testCaseResultData.SuiteTitle = className;
-            testCaseResultData.SuiteId = className;
+            string inputString = $"{testResultSource.TestCase.DisplayName}-{testResultSource.TestCase.CodeFilePath}";
+            var SuiteName = ReporterUtils.CalculateSha1Hash(inputString);
+            testCaseResultData.SuiteTitle = SuiteName;
```

Comment:
SuiteTitle shouldnt be hash value, only suiteId need to be hash

Reply:
how about just testCaseResultData.SuiteId = ReporterUtils.CalculateSha1Hash(className);;